tokio-reactor/src/lib.rs

```diff
@@ -32,38 +32,44 @@
 //! [`PollEvented`]: struct.PollEvented.html
 //! [reactor module]: https://docs.rs/tokio/0.1/tokio/reactor/index.html
 
-pub(crate) mod background;
+macro_rules! ready {
```

Comment:
This macro is available in [tokio-futures](https://github.com/tokio-rs/tokio/blob/std-future/tokio-futures/src/macros.rs#L3) if you want to use it there. I think the crate is already being pulled in via another dep.